src/utils/utils.js

```diff
@@ -281,7 +281,9 @@ export const getconstructorParams = (abiConstructor, state, vals, crowdsaleNum)
                 case "_oneTokenInWei": {
                   //params.vals.push(state.pricingStrategy[crowdsaleNum].rate);
                   //params.vals.push(state.web3.toWei(1/state.pricingStrategy[crowdsaleNum].rate/10**state.token.decimals, "ether"));
-                  params.vals.push(state.web3.utils.toWei(1/state.pricingStrategy[crowdsaleNum].rate, "ether"));
+                  
+                  let oneTokenInETH = roundToDecimals(1000000000000000000, 1/state.pricingStrategy[crowdsaleNum].rate)
```

Comment:
Extract the 1/state.pricingStrategy[crowdsaleNum].rate to a variable or function call